api/src/main/java/com/google/sps/Application.java

```diff
@@ -10,6 +10,17 @@
 
 public class Application {
   public static void main(String[] args) {
+    try {
+      Firebase.init();
+    } catch (Exception e) {
+      throw new RuntimeException(e);
```

Comment:
What's the point of catching an exception just to throw it again

Reply:
Is this not the same concept as wrapping exceptions?

Reply:
Why do you need to wrap in this case

Reply:
I was under the impression when we talked that it was just better practice

Reply:
Reason for rewrapping previously was because in terms of that API, the client sees it as an IOException since they are only caring about input-output. I don't think there is such thing as client api here